base/.eslintrc.json

```
@@ -48,6 +48,40 @@
         "allow": ["^UNSAFE_"]
       }
     ],
+    "import/order": [
+      1,
+      {
+        "newlines-between": "always",
+        "pathGroups": [
+          {
+            "group": "builtin",
+            "pattern": "^[@a-zA-Z]",
+            "position": "before"
+          },
+          {
+            "group": "external",
+            "pattern": "^[@a-zA-Z]",
+            "position": "before"
+          },
+          {
+            "group": "internal",
+            "pattern": "^[^@a-zA-Z]",
+            "position": "after"
+          },
+          {
+            "group": "parent",
+            "pattern": "^[^@a-zA-Z]",
+            "position": "after"
+          },
+          {
+            "group": "sibling",
+            "pattern": "*",
+            "position": "after"
+          }
+        ]
```

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array

Looks like this can be simplified a lot, and I think the current settings will group a lot of stuff separately where previously they were combined.

I believe the below config should match our existing tslint rules.

```suggestion
        "groups": [
          ["builtin", "external"],
          ["internal", "parent", "sibling", "index"],
          ["unknown"]
        ],
        "pathGroups": [
          {
            "group": "internal",
            "pattern": "^"
          }
        ]
```

---

@JakeSidSmith import/order rules have been updated.

---

https://github.com/benmosher/eslint-plugin-import/issues/1644